repo_health/index/static/client/src/components/base-component/base-stats.controller.js

```diff
@@ -0,0 +1,21 @@
+/*
+* base-stats.controller.js - (C) Copyright - 2017
+* This software is copyrighted to contributors listed in CONTRIBUTIONS.md.
+*
+* SPDX-License-Identifier: MIT
+*
+* Author(s) of this file:
+* J.Harding
+*
+* Base stats controller to handle an error state if no repos are returned for an owner login and name request
+*/
+
+
+export default class BaseStatsController {
+
+    constructor ($state, Urls) {
```

Comment:
The component used `$filter` and it is very confusing.  I will write service to use with the components and use `$resolve` to get ride the controller inheritance.